### PR TITLE
enabling instance can have volume in tests

### PR DESCRIPTION
**Pull request by cp16net**
fix the tests for volumes

### CODE REVIEW

**Comment by rnirmal** [comment]
+1

File: tests/integration/tests/api/instances.py [hunk 1]
```diff
@@ -241,6 +241,7 @@ class CreateInstance(unittest.TestCase):
 
     def test_instance_size_too_big(self):
         if test_config.values['reddwarf_can_have_volume']:
+            raise SkipTest("Wrong assertion because we have no volume limit")
```
Comment (ed-) [inline tests/integration/tests/api/instances.py:249]: No limit you say? :suspect: Okay...for now.

**Comment by sacharya** [comment]
+1

**Comment by TimSimpsonR** [comment]
+1

**Comment by ed-** [comment]
+1

**Pull request merged**